javascript/src/main/components/Statistics/FullCourseTable.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+import Badge from 'react-bootstrap/Badge'
+import Container from 'react-bootstrap/Container'
+
+const white = {
+  backgroundColor: '#ffffff'
+};
+
+const d2d4d7 = {
+  backgroundColor: '#d2d4d7'
+}
+
+const alternate = (cell, row, rowIndex, colIndex) => {
+  if (rowIndex % 2 === 0) 
+    return { backgroundColor: '#e9ecef' };
+  else
+    return { backgroundColor: '#edf0f2' };
+};
```

Comment:
Same... maybe 
```
const oddRows = '#e9ecef';
const evenRows = '#edf0f2';
```
And put all of these colors into a single JavaScript file that is imported, and then used everywhere these colors are needed in the app?

Reply:
Updated!